app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -39,6 +49,28 @@ protected void onCreate(Bundle savedInstanceState) {
 
         View infoCard = findViewById(R.id.info_card);
         swipeableInfoCard = BottomSheetBehavior.from(infoCard);
+
+        getApplication().getApplicationContext().deleteDatabase(AppDatabase.DB_NAME);
```

Comment:
why are you deleting it?

Reply:
deletes the db before loading the new stuff, eventually, we will be able to prepopulate the db with a .db file, but right now our stuff is in json. we still have to do more data collection, and that will be easier to do by putting it in json. so at the beginning of the application, every time, we import the json and then save it to the db, so we have to delete what was previously in the db. this should not go to prod